src/Okta.DNX.OAuth.ResourceServer/Models/TodoRepository.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace Okta.DNX.OAuth.ResourceServer.Models
+{
+    public class TodoRepository : ITodoRepository
+    {
+        private static ConcurrentDictionary<string, TodoItem> _todos =
```

Comment:
I don't know if it's consistent with other .Net projects, as far as I know, field and property in a class is UpperCamelCase, private or public.

also since it's constructed here, you could use `readonly` I think.